### PR TITLE
Docker compose read env file

### PR DESCRIPTION
**Pull request by adamalfredsson**
* Allow empty password
* Apparently the configuration took offense in using `root` value for `DB_USER`
* Read environment from env file

---

Tested on devices

- [ ] Desktop 💻
- [ ] Mobile 📱

Tests

- [ ] All tests are running ✔️
- [ ] Test are updated 🧪
- [ ] Code Review 👩‍💻
- [ ] QA 👌

Checkpoints

*Check these to flag for a more thurough review, as they could be potentially breaking changes*

- [ ] Packages updated
- [ ] Other infrastructure updated (such as node version or similar)

⏲️ Time spent on CR:

⏲️ Time spent on QA:

### CODE REVIEW

**Comment by adamalfredsson** [comment]
Build fixed in #580

**Comment by lorentzlasson** [comment]
> Apparently the configuration took offense in using root value for DB_USER

Hm, thought the root user should be created by default :thinking: Did you try having something else as DB_USER and then try connect with root/DB_PASS?

File: docker-compose.yml [hunk 1]
```diff
@@ -13,8 +13,9 @@ services:
     environment:
       MYSQL_DATABASE: ${DB_NAME}
       MYSQL_ROOT_PASSWORD: ${DB_PASS}
-      MYSQL_USER: ${DB_USER}
-      MYSQL_PASSWORD: ${DB_PASS}
+      MYSQL_ALLOW_EMPTY_PASSWORD: 1
+    env_file:
+      - .env
```
Comment (lorentzlasson) [inline docker-compose.yml:18]: The `.env` file is loaded by default I think

**Comment by adamalfredsson** [comment]
> > Apparently the configuration took offense in using root value for DB_USER
> 
> Hm, thought the root user should be created by default 🤔 Did you try having something else as DB_USER and then try connect with root/DB_PASS?

I'm confused now, unable to reproduce the issue I encountered. Will revert these changes until it might appear again.

File: docker-compose.yml (hunk 1 above)
Comment (adamalfredsson) [reply docker-compose.yml:19]: That would be great, I'm unable to find documentation about this though :/

Comment (adamalfredsson) [reply docker-compose.yml:18]: Seems like it does though, but maybe it's still nice to be explicit?

Comment (lorentzlasson) [reply docker-compose.yml:18]: Sure :+1:

**Review by lorentzlasson — APPROVED** [review]

**Pull request merged**